common/security.py

```
@@ -0,0 +1,55 @@
+        settings.ALGORITHM
+    )
+    return encoded_jwt
+
+def decode_access_token(token: str) -> str | None:
+    try:
+        payload = jwt.decode(token,
+                             settings.SECRET_KEY,
+                             algorithms=[settings.ALGORITHM]
+                             )
+        email: str= payload.get("sub")
+        return email
```

Ruff (0.14.11)

Consider moving this statement to an `else` block

(TRY300)